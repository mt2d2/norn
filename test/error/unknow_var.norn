def main(): Void

	println(unknown)

end
