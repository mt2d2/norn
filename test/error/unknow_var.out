undefined function: println:DEFAULT_TYPE
